### DISCUSSION TITLE
x-codeSamples + autogenerated samples

### DISCUSSION THREAD

**Discussion by marclave** [scalar/scalar#2256] [General]
Currently when we have x-codeSamples, we make that the only option inside of the request examples. however we dont show the autogenerated samples that you can select from the introduction section

i wanted to open a discussion on if we should:
- hide all examples by default then the user can select clients
- show all clients by default but add the example as the primary option when showing examples

https://docs.val.town/openapi#tag/vals/POST/v1/eval

can see this here

**Comment by tmcw** [comment]
> hide all examples by default then the user can select clients

I think this would be idea, something like an explicit list of which clients to show, including codeSamples, could be good, like `examples={['codeSamples', 'python', 'node', 'curl']}`

**Comment by hanspagel** [comment]
Working on it :)

PR https://github.com/scalar/scalar/pull/2365
